src/tcgwars/logic/impl/gen4/MajesticDawn.groovy

```diff
@@ -177,15 +177,19 @@ public enum MajesticDawn implements LogicCardInfo {
           resistance F, MINUS20
           pokePower "Freezing Screech", {
             text "Once during your turn, when you put Articuno from your hand onto your Bench, you may flip a coin. If heads, choose 1 of the Defending Pokémon. That Pokémon is now Paralyzed."
-            actionA {
+            onActivate { 
+              if(it==PLAY_FROM_HAND && confirm("Use Freezing Screech?")) {
+                powerUsed()
+                flip {apply PARALYZED}
+              }
             }
           }
           move "Blizzard", {
             text "60 damage. Flip a coin. If heads, this attack does 10 damage to each of your opponent’s Benched Pokémon. If tails, this attack does 10 damage to each of your Benched Pokémon."
             energyCost W, W, C
-            attackRequirement {}
             onAttack {
-              damage 0
+              damage 60
+              flip {my.bench.each {damage 10, it}} {opp.bench.each {damage 10, it}}
```

Comment:
The structure of `flip` if you want to write a closure for the tails result is:

`flip 1, { ... }, {...}`